gcc/testsuite/rust/execute/torture/builtin_macro_env.rs

```diff
@@ -0,0 +1,21 @@
+// { dg-output "VALUE\n" }
```

Comment:
```suggestion
// { dg-output "VALUE\n" }
// { dg-set-compiler-env-var ENV_MACRO_TEST "VALUE" }
```
... And add this. Taken directly from https://gcc.gnu.org/onlinedocs/gccint/Directives.html for future references

Reply:
Thank you for your help. My solution was awkward. I knew there had to be a better one :)